python/ray/serve/_private/controller.py

```diff
@@ -207,6 +218,29 @@ async def __init__(
             description="The number of times that controller has started.",
         ).inc()
 
+    def reconfigure_system_logging_config(self, logging_config: LoggingConfig):
+        if (
+            self.logging_config
+            and self.logging_config.version == logging_config.version
+        ):
+            return
+        self.kv_store.put(LOGGING_CONFIG_CHECKPOINT_KEY, pickle.dumps(logging_config))
+        self.logging_config = logging_config
+
+        self.long_poll_host.notify_changed(
+            LongPollNamespace.SYSTEM_LOGGING_CONFIG,
+            logging_config,
+        )
+        configure_component_logger(
+            component_name="controller",
+            component_id=str(os.getpid()),
+            logging_config=logging_config,
+        )
+        logger.debug(
+            "Configure the serve controller logger "
+            f"with logging config: {self.logging_config}"
+        )
```

Comment:
Would make sure this only logs when the config is changed then make it info-level (it's useful)
```suggestion
        logger.debug(
            f"Serve controller logging config updated to: {self.logging_config}."
        )
```